Dockerfile

```diff
@@ -60,7 +69,6 @@ RUN set -eux; \
     sudo chmod +x /usr/local/bin/datadog-ci;\
```

Comment:
Where does this file comes from?

Reply:
@PerfectSlayer https://github.com/DataDog/datadog-ci
We use it to upload test results to CI Visibility: https://github.com/DataDog/dd-trace-java/blob/b68e2883a3f7e5ca9c97731645a72fad22e983ff/.circleci/upload_ciapp.sh#L17